pkg/controller/storagecluster/reconcile.go

```diff
@@ -33,6 +34,9 @@ type ReconcileStrategy string
 // StorageClassProvisionerType is a string representing StorageClass Provisioner. E.g: aws-ebs
 type StorageClassProvisionerType string
 
+// DeviceTypes is a string representing DeviceType
```

Comment:
New structs shouldn't be plural.

```suggestion
// DeviceType is a string representing a storage device type
```

Reply:
done